background_scripts/completion_search.coffee

```diff
@@ -48,12 +48,19 @@ CompletionSearch =
     xhr = new XMLHttpRequest()
     xhr.open "GET", url, true
     xhr.timeout = 2500
-    xhr.ontimeout = xhr.onerror = -> callback null
-    xhr.send()
+    finished = 0
```

Comment:
On inspection, it seems to me that callback should never be called twice. The only way it can be called twice is if the definition of the ontimeout and onerror events overlap (meaning onerror fires when a timeout occurs).

If that's true, then the simpler fix to this problem is to just remove the ontimeout handler, assuming onerror is the more generic event.